app/models/tweet.rb

```diff
@@ -1,3 +1,7 @@
 class Tweet < ActiveRecord::Base
   belongs_to :user
+  default_scope -> { order(created_at: :desc) }
```

Comment:
not sure if the arrow is necessary and/or might be older syntax?

http://api.rubyonrails.org/classes/ActiveRecord/Scoping/Default/ClassMethods.html